src/screens/docs/components/victory-axis/ecology.md

```diff
@@ -2,6 +2,10 @@
 
 VictoryAxis renders a single axis which can be used on its own or composed with [VictoryChart].
 
+```playground
+<VictoryAxis tickValues={["apples", "bananas", "oranges"]}/>
```

Comment:
I think this is a confusing example. People are frequently confused about the difference between `tickValues` and `tickFormat` and I think this example muddies the waters.